addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -68,6 +68,9 @@
                 background-image: url("/web_editor/shape/#{$module}/#{$shape}.svg?#{str-slice($url-params, 2)}");
                 background-position: map-get($style, 'position');
                 background-size: map-get($style, 'size');
+                @if map-get($style, 'repeat-x') {
+                    background-repeat: repeat-x;
+                }
                 @if map-get($style, 'repeat-y') {
                     background-repeat: repeat-y;
                 }
```

Comment:
I'll handle the case where both repeat are true. This is actually easy to write using the two values syntax:
```scss
                background-repeat:
                    if(map-get($style, 'repeat-x'), repeat, no-repeat)
                    if(map-get($style, 'repeat-y'), repeat, no-repeat);
```